try {
    alert('Action completed!');
    return x - y;
    data.push(value);
} catch (error) {
    console.error("Error:", error);
}